layouts/partials/menu.html

```diff
@@ -34,5 +49,13 @@
         <li><a href="{{ .URL }}">{{ .Name }}</a></li>
       {{ end }}
     {{ end }}
+    {{ if $.Site.Params.showLanguageSelector }}
+    <hr />
+        {{ range $.Site.Home.AllTranslations }}
+          <li class="list-inline-item">
```

Comment:
Does `list-inline-item` class do something? 🤔

Reply:
What about this?

Reply:
Oh forgot these. Not used anymore I will remove them.